### ISSUE TITLE
Unexpected behavior when pressing the delete / backspace key in AMP Story editor  

### ISSUE THREAD

**Issue by spacedmonkey** [ampproject/amp-wp#3280]
When a block is selected and delete key is pressed, sometimes a block is not remove / deleted. This leads to press the delete key multiple times, that then deletes the page block. Generally the bahavior of the delete key in the AMP Story editor, should be handled better for accessibility reasons.

**Comment by spacedmonkey** [comment]
See this screenshot that show the errors. 

https://www.youtube.com/watch?v=hvf0jOYe4Dg&feature=youtu.be

This screenshot shows two different two different errors. 

1 .When blocks are selected, the delete and backspace key are being pressed and the block is not removed. 
1. When no block is selected and delete key is pressed, the whole page including of it's block is removed. But this only happens sometimes. It also means that the first page is removed, which is something that shouldn't be possible.

**Comment by spacedmonkey** [comment]
See this screenshot that show the errors.

https://www.youtube.com/watch?v=hvf0jOYe4Dg&feature=youtu.be

This screenshot shows two different two different errors.

1 .When blocks are selected, the delete and backspace key are being pressed and the block is not removed.
1. When no block is selected and delete key is pressed, the whole page including of it's block is removed. But this only happens sometimes. It also means that the first page is removed, which is something that shouldn't be possible.

**Comment by spacedmonkey** [comment]
See this screenshot that show the errors.

https://www.youtube.com/watch?v=hvf0jOYe4Dg&feature=youtu.be

This screenshot shows two different two different errors.

0. When blocks are selected, the delete and backspace key are being pressed and the block is not removed.
0. When no block is selected and delete key is pressed, the whole page including of it's block is removed. But this only happens sometimes. It also means that the first page is removed, which is something that shouldn't be possible.

**Comment by spacedmonkey** [comment]
See this screenshot that show the errors.

https://www.youtube.com/watch?v=hvf0jOYe4Dg&feature=youtu.be

This screenshot shows two different two different errors.

1. When blocks are selected, the delete and backspace key are being pressed and the block is not removed.
2. When no block is selected and delete key is pressed, the whole page including of it's block is removed. But this only happens sometimes. It also means that the first page is removed, which is something that shouldn't be possible.

**Comment by westonruter** [comment]
I just noticed this behavior today when testing. I was surprised that pressing delete would not remove a selected block. I had to use the context menu to remove the block. (I could have used the Gutenberg shortcut, but I don't have it memorized.)

**Comment by spacedmonkey** [comment]
Removing blocks / pages is not simple at the moment. I have created two related issues #3279 #3221

**Comment by spacedmonkey** [comment]
As a talking point, I have created a WIP pull request in #3400. It is still discovery stages at this point, but it is a workable solution.

**Comment by jauyong** [comment]
@spacedmonkey @westonruter 
I added different block types and only the text block failed to be deleted. Is this what you are experiencing as well? 

Although it looks like I'm struggling to delete the CTA, that was just me. I tested and it was pretty seamless. 


![3280](https://user-images.githubusercontent.com/5874851/67889579-7f52b800-fb25-11e9-9245-ca3ba2edf817.gif)
